packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -1,14 +1,20 @@
-import { BigNumber, Wallet, constants, utils } from 'ethers'
+import {
+  BigNumber,
+  Contract,
+  ContractFactory,
+  Wallet,
+  constants,
+  utils
+} from 'ethers'
 import { defineConfig } from 'cypress'
 import { StaticJsonRpcProvider } from '@ethersproject/providers'
 import synpressPlugins from '@synthetixio/synpress/plugins'
 import { TestWETH9__factory } from '@arbitrum/sdk/dist/lib/abi/factories/TestWETH9__factory'
-import { TestERC20__factory } from '@arbitrum/sdk/dist/lib/abi/factories/TestERC20__factory'
```

Comment:
what's the reason behind removing the usage of `TestERC20__factory`?

Reply:
`TestERC20__factory` was initially being used to deploy ERC20 token (`IArbToken`) provided by SDK but there are multiple issues in that - we can't use bigger amounts, approval is a bit messy. That's why we moved from this factory initially to WETH now, and now that we need a standard Erc20 token, we're creating an instance of our own.